### ISSUE TITLE
Pagination support?

### ISSUE THREAD

**Issue by killeriax** [pavankataria/SwiftDataTables#57]
Hi! As far as I understand, this library currently does not support pagination?

**Comment by pavankataria** [comment]
That's correct. However, this is a feature that's also in the pipeline. Right now there's full focus on allowing self sizing cells with the ability to provide custom views. Soon after this feature will come in. 

You can see in the codebase that work on pagination has already bled through ;)

**Comment by tranquangdat1397** [comment]
Has this feature been implemented yet?

**Comment by pavankataria** [comment]
Thanks for this feature request! I've migrated all feature requests to the new [voting page](https://swiftdatatables.pavankataria.com/vote) where you can upvote this and other features.

This helps prioritise what I should work on next based on community interest. Please vote if you'd like to see this implemented!

Closing this issue as feature requests are now tracked on the website.
